external-crates/move/documentation/book/src/unit-testing.md

```diff
@@ -12,29 +12,30 @@ annotations can be placed on a function with any visibility. Whenever a module o
 annotated as `#[test_only]` or `#[test]`, it will not be included in the compiled bytecode unless it
 is compiled for testing.
 
-## Testing Annotations: Their Meaning and Usage
+## Test Annotations
 
-Both the `#[test]` and `#[expected_failure]` annotations can be used either with or without
-arguments.
+The `#[test]` annotation can only be placed on a function with no parameters.
+This annotation marks the function as a test to be run by the unit testing harness.
 
-Without arguments, the `#[test]` annotation can only be placed on a function with no parameters.
-This annotation simply marks this function as a test to be run by the unit testing harness.
-
-```
+```move
 #[test] // OK
 fun this_is_a_test() { ... }
 
 #[test] // Will fail to compile since the test takes an argument
-fun this_is_not_correct(arg: signer) { ... }
+fun this_is_not_correct(arg: u64) { ... }
 ```
 
-A test can also be annotated as an `#[expected_failure]`. This annotation marks that the test is
-expected to raise an error. You can ensure that a test is aborting with a specific abort code by
-annotating it with `#[expected_failure(abort_code = <code>)]`, if it then fails with a different
-abort code or with a non-abort error the test will fail. Only functions that have the `#[test]`
-annotation can also be annotated as an #`[expected_failure]`.
+A test can also be annotated as an `#[expected_failure]`. This annotation marks
+that the test is expected to raise an error. There are a number of 
```

Comment:
The wrapping here seems a bit strange, do you have prettier installed?

Reply:
What is prettier? I just use `vip gq` in Vim... :p

Reply:
Looks like the formatting got messed up during a rebase :'(